MIDIchlorians/MIDIchlorians/Session.swift

```diff
@@ -52,6 +52,10 @@ class Session: Object {
         return sessionName
     }
 
+    func getPad(page: Int, indexPath: IndexPath) -> Pad {
```

Comment:
Cool. The method I've implemented in my branch takes in a page, row, and column. Is that ok?